packages/@react-spectrum/datepicker/test/DatePicker.test.js

```diff
@@ -2334,4 +2334,70 @@ describe('DatePicker', function () {
       });
     });
   });
+
+  describe('style', () => {
```

Comment:
in general, testing styles in a unit test is not helpful since it doesn't actually render, these are all perfect for chromatic though